package.json

```
@@ -54,6 +55,10 @@
 					"light": "resources/light/refresh.svg",
 					"dark": "resources/dark/refresh.svg"
 				}
+      },
+      {
+        "command": "ptt.favorite-board",
+        "title": "我的最愛看板"
```

```suggestion
        "title": "從我的最愛新增訂閱"
```

這裡改一下用詞。MVP 不想加這個功能是因為 upstream library 還沒法一次撈完全部我的最愛 😂

---

我以為我撈完了欸XD，原來是我的最愛太少了嗎